packages/lesswrong/lib/rateLimits/constants.ts

```diff
@@ -197,8 +180,7 @@ export const autoCommentRateLimits: ForumOptions<CommentAutoRateLimit[]> = {
     ...EA.COMMENTS
   ],
   LessWrong: [
-    ALL.COMMENTS.ONE_PER_EIGHT_SECONDS, 
```

Comment:
why get rid of this one?

Reply:
I predict that you will end up with a lot of duplicate comments this way. This is actually the "don't double post" rate limit.